source/reference/api/commit.txt

```diff
@@ -75,6 +73,13 @@ Before using the ``commit`` endpoint:
        When ``lagTimeSeconds`` is ``0``, the source and destination
        clusters are in a consistent state.
 
+  - ``state: "RUNNING"``
+  - ``canCommit: true``
+
+.. note::
+
+   .. include:: /includes/api/facts/multiple-mongosync-endpoints
```

Comment:
I think on all of the API endpoint pages it becomes confusing because it doesn't link to a multiple mongosync section that explains how to use it. Could we try to add something like that?